react-components/my-app/src/App.tsx

```diff
@@ -0,0 +1,17 @@
+import { Image } from './Image';
+import { Header } from './Header';
+import { Caption } from './Caption';
+import { Button } from './Button';
+import { Description } from './Description';
+
+export function App() {
+  return (
+    <>
+      <Header></Header>
```

Comment:
You should get in the habit of using self-closing tags. Many React developers get annoyed when they see code that doesn't use them and they think that the developer must have been a beginner! I want them thinking of you in only glowing terms when they read your code!